apps/project/graphql/inputs.py

```diff
@@ -27,6 +27,16 @@ class TileServerCommonConfigInput: ...
 class ProjectTileServerConfigInput: ...
 
 
+@strawberry.experimental.pydantic.input(model=completeness_project.VectorTileServerConfig, all_fields=True)
+class ProjectVectorTileServerConfigInput: ...
```

Comment:
To avoid name clash in the future, let's add `Completeness` prefix

Reply:
This is a generic input so maybe we should move this to utils later.